### PR TITLE
New package: libaccounts-qt-1.14, libaccounts-glib-1.21

### PR DESCRIPTION
**Pull request by subsonik**
libaccounts-qt is a dependency of kaccounts-integration, part of KDE Plasma 5. libaccounts-glib is a dependency of libaccounts-qt.

### CODE REVIEW

File: srcpkgs/libaccounts-qt/template [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+libaccounts-qt-devel_package() {
+	short_desc+=" - development files"
+	depends="${sourcepkg}>=${version}_${revision}"
+	pkg_install() {
+		vmove usr/include
+		vmove usr/lib/*.so
+		vmove usr/lib/pkgconfig
+		vmove usr/share/doc/accounts-qt
```
Comment (Vaelatern) [inline srcpkgs/libaccounts-qt/template]: Unless the docmentation is only necessary for the development package, the documentation probably should go in the main package.

Comment (subsonik) [inline srcpkgs/libaccounts-qt/template]: The documentation consists of a brief introduction and then just class/API references. I presume this should be a part of the -devel package? Sorry if not, I'm new to this!

Comment (Vaelatern) [inline srcpkgs/libaccounts-qt/template]: In this case, that makes sense.

**Comment by subsonik** [comment]
Updated, thanks for your feedback.

**Comment by asergi** [comment]
Please rename the `libaccounts-qt` package to `libaccounts-qt5`.

**Comment by subsonik** [comment]
I've renamed the package but the doc folder is still called `libaccounts-qt`, which is hard-coded into the Makefile, I can knock up a patch to fix this if needed.

File: common/shlibs [hunk 2]
```diff
@@ -2412,3 +2412,5 @@ libcmark.so.0 cmark-0.24.1_1
 libspotify.so.12 libspotify-12.1.51_1
 libwiredtiger-2.8.0.so wiredtiger-2.8.0_1
 libvidstab.so.0.9 libvidstab-0.98b_1
+libaccounts-glib.so.0 libaccounts-glib-1.21_1
+libaccounts-qt5.so.1 libaccounts-qt-1.14_1
```
Comment (asergi) [inline common/shlibs]: `libaccounts-qt5.so.1 libaccounts-qt5-1.14_1`

File: srcpkgs/libaccounts-qt5/template [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Template file for 'libaccounts-qt5'
+
+pkgname="libaccounts-qt5"
+_pkgname="libaccounts-qt"
```
Comment (asergi) [inline srcpkgs/libaccounts-qt5/template]: You can drop `_pkgname` var.

File: srcpkgs/libaccounts-qt5/template [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+build_style=qmake
+short_desc="Client library to the accounts database, with a Qt5-based API"
+maintainer="John Rowley <enterthevoid@codesector.co>"
+license="LGPL-2.1"
+homepage="https://gitlab.com/accounts-sso/libaccounts-qt"
+distfiles="${homepage}/repository/archive.tar.gz?ref=VERSION_${version}>${pkgname}-${version}.tar.gz"
+checksum="53d2e295b26295166db5c16333c91e32a599b36c2eee296b803ce39d6a99a8d9"
+hostmakedepends="pkg-config doxygen"
+makedepends="libaccounts-glib-devel qt5-devel"
+wrksrc="${_pkgname}-VERSION_${version}-a34ca4b6d250529c900b0382559553b6e5885918"
```
Comment (asergi) [inline srcpkgs/libaccounts-qt5/template]: wrksrc="${pkgname/5/}-VERSION_${version}-a34ca4b6d250529c900b0382559553b6e5885918"

**Comment by asergi** [comment]
> I've renamed the package but the doc folder is still called libaccounts-qt, which is hard-coded into
> the Makefile, I can knock up a patch to fix this if needed.

Not an issue for now.

**Comment by Vaelatern** [comment]
@subsonik Please rebase on top of master and push again.

**Comment by Vaelatern** [comment]
The rule within Void is that for every template altered, there should be one commit. Those commits should be named as directed by the end of the [Manual (link)](https://github.com/voidlinux/void-packages/blob/master/Manual.md#contributing-via-git).

Only one PR, please, unless you are told otherwise. Unrelated features, such as if you fixed something in `firefox` and added a new spotify client, should go in separate PRs. If you wanted to package something, and it depended on a different package not yet in Void, you would put both of those package additions in the **same** PR.

**Pull request closed without merging**